turbopack/crates/turbo-tasks/src/native_function.rs

```diff
@@ -268,13 +250,12 @@ impl NativeFunction {
         )
     }
 
-    pub fn resolve_span(&'static self, persistence: TaskPersistence) -> Span {
-        let flags = match persistence {
-            TaskPersistence::Persistent => "",
-            TaskPersistence::Transient => "transient",
-            TaskPersistence::Local => "local",
-        };
-        tracing::trace_span!("turbo_tasks::resolve_call", name = self.name, flags = flags)
+    pub fn resolve_span(&'static self) -> Span {
+        tracing::trace_span!(
+            "turbo_tasks::resolve_call",
+            name = self.name,
+            flags = "local"
```

Comment:
```suggestion
```

I think we could omit that as all resolve calls are local tasks. Saves a bit of data in the trace